diff --git a/suitesparse_graphblas/__init__.py b/suitesparse_graphblas/__init__.py
index 7569034..e356d6a 100644
--- a/suitesparse_graphblas/__init__.py
+++ b/suitesparse_graphblas/__init__.py
@@ -6,7 +6,7 @@ from ._graphblas import ffi, lib  # noqa
 import struct
 import platform
 
-_is_osx_arm64 = platform.machine() == "arm64"
+_is_osx_arm64 = True or platform.machine() == "arm64"
 _c_float = ffi.typeof("float")
 _c_double = ffi.typeof("double")
 
diff --git a/suitesparse_graphblas/build.py b/suitesparse_graphblas/build.py
index e8a18ee..c646598 100644
--- a/suitesparse_graphblas/build.py
+++ b/suitesparse_graphblas/build.py
@@ -5,7 +5,7 @@ import platform
 from cffi import FFI
 
 is_win = sys.platform.startswith("win")
-is_arm64 = platform.machine() == "arm64"
+is_arm64 = True or platform.machine() == "arm64"
 thisdir = os.path.dirname(__file__)
 
 ffibuilder = FFI()
